arangod/RestHandler/RestViewHandler.cpp

```diff
@@ -83,6 +83,25 @@ void RestViewHandler::getView(std::string const& nameOrId, bool detailed) {
     return;
   }
 
+  // skip views for which the full view definition cannot be generated, as per https://github.com/arangodb/backlog/issues/459
+  try {
+    arangodb::velocypack::Builder viewBuilder;
+
+    viewBuilder.openObject();
+
+    auto res = view->toVelocyPack(viewBuilder, true, false);
```

Comment:
isn't there a better way of doing this ?

Reply:
Can't you use `velocypack::Builder::removeLast()` ?

Reply:
I am unsure how removeLast() would be applicable in this situation. According the the description it only applies to the last value and only of an unclosed object, neither of which is guaranteed.